.github/workflows/build_and_test.yml

```diff
@@ -352,4 +389,4 @@ jobs:
       shell: bash
       run: |
         lua .ci/luacov-stats-aggregator.lua "luacov-stats-out-" "luacov.stats.out" ${{ github.workspace }}/
-        awk '/Summary/,0' luacov.report.out
+        awk -v RS='Summary\n=+\n' 'NR>1{print $0}' luacov.report.out
```

Comment:
Care to elaborate on the effect of this change?

Reply:
the newer version prints the content starting from the **last** occurrence of the word "Summary" in the report file. The previous was printing starting from the first occurrence, so if there happened to be any "Summary" in the source, it would print starting from there, rather than the actual Summary.